context/app/markdown/docs/metadata.md

```diff
@@ -36,43 +36,13 @@ Donor Specification
 - [PAS (histological stain)](/docs/assay/pas)
   - [Spec](https://github.com/hubmapconsortium/ingest-validation-tools/tree/master/docs/stained)
   - Contact: [Jeffrey Spraggins](mailto:jeff.spraggins@Vanderbilt.Edu)
-- [RNAseq (bulk)](/docs/assay/rnaseq)
+- [RNAseq](/docs/assay/rnaseq)
   - [Spec](https://github.com/hubmapconsortium/ingest-validation-tools/tree/master/docs/bulkrnaseq)
   - Contact: [Stephanie Nevins](mailto:snevins@stanford.edu)
-- [sciATAseq](/docs/assay/atacseq)
-  - [Spec](https://github.com/hubmapconsortium/ingest-validation-tools/tree/master/docs/scatacseq)
```

Comment:
Note that under ATACseq above, you have
```
  - [Spec](https://github.com/hubmapconsortium/ingest-validation-tools/tree/master/docs/bulkatacseq)
  - Contact: [Blue Lake](mailto:b1lake@eng.ucsd.edu)
```
which is different than what you're deleting here: You're not just removing redundant information.

Reply:
I have placed the contacts into the relevant sections of the document. There's one document for all RNAseq. There's one document for all ATACseq. There's one document for all LC-MS.
All of the RNAseq links take you to the same RNAseq doc. Likewise for ATACseq and LC-MS. They (Peter & Melissa??) created a link for each subtype of RNAseq. Likewise for RNAseq and LC-MS. The single document for RNAseq contains all of the descriptions for all of the RNAseq subtypes.

Reply:
I have to correct that. I put the contact links into the ATAcseq doc but have to fix that one. Will review.

Reply:
No worries; ping me again when it's ready to go.